plugins/processors/gpuattributes/processor_test.go

```diff
@@ -19,93 +19,170 @@ func TestProcessMetrics(t *testing.T) {
 	ctx := context.Background()
 
 	testcases := map[string]struct {
-		resource string
-		metrics  pmetric.Metrics
-		wantCnt  int
-		want     map[string]string
+		resource          string
+		metrics           pmetric.Metrics
+		wantCnt           int
```

Comment:
nit - can we rename wantCnt? It not completely obvious what is it used for (something like wantDatapointsCnt is good)

Reply:
agree, updated.